src/test/java/models/testDoubleDown.java

```diff
@@ -0,0 +1,21 @@
+package models;
+
+import org.eclipse.jetty.server.Authentication;
+import org.junit.Test;
+
+/**
+ * Created by Terrance on 3/8/2016.
+ */
+public class testDoubleDown {
+    @Test
+    public void testDoubleDown(){
+        Player P = new Player();
+            P.setBet(100);
+            P.doubleDown(50);
+        AssertNotEquals(250, P.setBet);
+    }
+
```

Comment:
Please explain how this worked since you code does not reflect a correct output.